config/config.go

```diff
@@ -678,23 +678,23 @@ func printConfig(name string, p telegraf.PluginDescriber, op string, commented b
 	if commented {
 		comment = "# "
 	}
-	fmt.Printf("\n%s# %s\n%s[[%s.%s]]", comment, p.Description(), comment, op, name)
```

Comment:
This is needed if there's no description. Otherwise we'll get `# no configuration` without context